tools/pylint-extensions/azure-pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1384,7 +1384,8 @@ def check_return(self, node):
         function_decorators = node.decoratornames()
         try:
             returns = next(node.infer_call_result()).as_string()
-            if returns == "None":
+            # If returns None, or raises an error ignore
+            if returns == "None" or returns == astroid.Uninferable:
```

Comment:
we call as_string() on L1386, is `returns` always string or does the == `astroid.Uninferable` comparison work?

Reply:
Calling as_string() on Uninferable returns Uninferable